src/cli.rs

```diff
@@ -15,6 +23,34 @@ pub struct Rover {
     #[structopt(long = "log", short = "l", global = true, default_value = DEFAULT_LEVEL, possible_values = &LEVELS, case_insensitive = true)]
     #[serde(serialize_with = "from_display")]
     pub log_level: Level,
+
+    #[structopt(skip)]
+    #[serde(skip_serializing)]
+    pub env_store: RoverEnv,
+}
+
+impl Rover {
+    pub(crate) fn get_config(&self) -> Result<Config> {
```

Comment:
```suggestion
    pub(crate) fn get_rover_config(&self) -> Result<Config> {
```
lots of configs in this file! I think it might be better to super extra specific that this is rover's env var config storage (right?), aka `rover_config`.